backend/infrahub/api/schema.py

```diff
@@ -256,19 +257,23 @@ async def load_schema(
                 ).value,
             ),
             branch=branch,
-        ):
-            raise PermissionDeniedError("You are not allowed to manage the schema")
+        )
+        if branch.name in (GLOBAL_BRANCH_NAME, registry.default_branch):
+            has_permission &= await permission_backend.has_permission(
```

Comment:
I think in the case of multiple backends, if the first returned `False` and the second returned `True`, this would result in `has_permission=False` b/c the `False` would make every subsequent `has_permission &=` be `False`